mmv1/products/artifactregistry/api.yaml

```
@@ -90,7 +90,14 @@ objects:
     - !ruby/object:Api::Type::String
       name: format
       description: |-
-        The format of packages that are stored in the repository.
+        The format of packages that are stored in the repository. You can only create alpha formats if you are a member of the [alpha user group](https://cloud.google.com/artifact-registry/docs/supported-formats#alpha-access).
```

We generally try to keep the line lengths shortened for the yaml files, so you could add a newline somewhere like this:
```suggestion
        The format of packages that are stored in the repository. You can only create
        alpha formats if you are a member of the [alpha user group](https://cloud.google.com/artifact-registry/docs/supported-formats#alpha-access).
```

---

Thanks!